### PR TITLE
Added support for setting the rcpreq parameter in the request.

### PR DESCRIPTION
**Pull request by voldern**

### CODE REVIEW

File: lib/pswincom/request.rb [hunk 1]
```diff
@@ -37,6 +37,7 @@ module PSWinCom
         m.TTL args[:TTL] if args[:TTL]
         m.TARIFF args[:tariff] if args[:tariff]
         m.SERVICECODE args[:servicecode] if args[:servicecode]
+        m.RCPREQ((args[:rcpreq] == true ? 'Y' : 'N')) if args[:rcpreq]
```
Comment (tormaroe) [inline lib/pswincom/request.rb]: Hmmm, can this ever become 'N' ??

Comment (voldern) [inline lib/pswincom/request.rb]: Hmm, good question. If args[:rcpreq] is set to a string it will become 'N'. It will only become 'Y' when the value is set to the boolean true. The reason I did it this way is so that a string like 'FALSE', or 'N' won't set RCPREQ to 'Y'.

The question is then if this is wanted behavior, or if should just set it to true as long as its not nil/false or even just pass along the args[:rcpreq] variable.
Another way to do it could be throw an exception when its set to something other then true/false/nil and just set it to 'Y' if args[:rcpreq] is true.

I think my last suggestion - throwing an ArgumentError - is probably the best way to do it. It prevents args[:rcpreq] from being set to an invalid value, and prevents strings like 'FALSE', 'N' etc from being parsed as true.

Comment (tormaroe) [inline lib/pswincom/request.rb]: I agree, making sure it's one of  true/false/nil sounds appropriate. Will you fix this and do another pull request?

Comment (voldern) [inline lib/pswincom/request.rb]: Oops, when I wrote that last comment my mined lapsed into PHP-mode and with it its unusual take on the double equal sings. In retrospect I don't know what I was thinking when I wrote the original code either. It should be enough to just do this:
`m.RCPREQ 'Y' if args[:rcpreq] == true`

I've added a new commit that does this.
